plugins/GSdx/GSSettingsDlg.cpp

```diff
@@ -738,6 +746,7 @@ void GSHacksDlg::OnInit()
 	EnableWindow(GetDlgItem(m_hWnd, IDC_ROUND_SPRITE), !native);
 	EnableWindow(GetDlgItem(m_hWnd, IDC_SPRITEHACK_TEXT), !native);
 	EnableWindow(GetDlgItem(m_hWnd, IDC_ROUND_SPRITE_TEXT), !native);
+	EnableWindow(GetDlgItem(m_hWnd, IDC_OFFSETHACK_TEXT), !native);
```

Comment:
Please add this directly below ``EnableWindow(GetDlgItem(m_hWnd, IDC_OFFSETHACK), !native);``